config/deployer.default.json

```diff
@@ -4,6 +4,7 @@
   "AppInstanceType": "c5.xlarge",
   "AgentInstanceCount": 2,
   "AgentInstanceType": "t3.xlarge",
+  "EnableAgentLogs": true,
```

Comment:
Maybe rename this to `EnableAgentFullLogs` or something like that because logs are enabled even if that is set to false, just differently.